# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/nvi/no-machine-setting.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile~	Thu Oct 28 09:15:53 1999
+++ ./Makefile	Thu Oct 28 09:18:40 1999
@@ -6,7 +6,7 @@
 #
 
 CC		= gcc
-CFLAGS	= -O6 -fomit-frame-pointer -m486 -s -pipe
+CFLAGS	= -O6 -fomit-frame-pointer -s -pipe
 
 BINDIR		= /usr/bin
 MANPREFIX	= n
